pkg/host/internal/network/network.go

```diff
@@ -23,6 +24,8 @@ import (
 	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/vars"
 )
 
+var ManifestsPath = "./bindata/manifests"
```

Comment:
lets put this in consts

Reply:
It's not needed anymore, so I deleted it